src/api/status/src/js/feed-stats.js

```diff
@@ -1,11 +1,15 @@
 const { fetch } = require('@senecacdot/satellite');
 
+const { POSTS_URL, POSTS_PORT } = process.env;
+
 module.exports = async function getFeedCount() {
   let feedCount = 0;
   try {
-    const data = await fetch(`${process.env.API_URL}/v1/posts/feeds`, {
-      method: 'HEAD',
-    });
+    const API_URL = POSTS_URL
+      ? `${POSTS_URL}:${POSTS_PORT}/feeds`
+      : `http://${process.env.API_HOST}/v1/posts/feeds`;
```

Comment:
You should be able to use `POSTS_URL` in the backend without having to play with it.  For local dev, change what's defined in `env.local`.